PennMobile/New Group/PAC Code/PacCodeViewController.swift

```diff
@@ -58,22 +61,59 @@ class PacCodeViewController : UIViewController, ShowsAlert, IndicatorEnabled {
         pacCodeHStack.centerXAnchor.constraint(equalTo: view.safeAreaLayoutGuide.centerXAnchor).isActive = true
     }
     
-    func setupPacCodeInfoLabel() {
-        let pacCodeInfoLabel = UILabel()
-        pacCodeInfoLabel.textColor = .labelPrimary
-        pacCodeInfoLabel.font = UIFont.systemFont(ofSize: 35)
-        pacCodeInfoLabel.text = "Your PAC Code is"
+    func setupPacCodeSecurityInfoLabel() {
+        pacCodeSecurityInfoLabel.textColor = .labelSecondary
+        pacCodeSecurityInfoLabel.font = UIFont.systemFont(ofSize: 16)
+        pacCodeSecurityInfoLabel.text = "After being fetched from Campus Express, your PAC code is stored in your device’s Secure Enclave.\n\nIt requires your authentication to view each time and never leaves your device."
         
-        view.addSubview(pacCodeInfoLabel)
-        pacCodeInfoLabel.translatesAutoresizingMaskIntoConstraints = false
-        pacCodeInfoLabel.bottomAnchor.constraint(equalTo: pacCodeHStack.topAnchor, constant: -30).isActive = true
-        pacCodeInfoLabel.centerXAnchor.constraint(equalTo: view.safeAreaLayoutGuide.centerXAnchor).isActive = true
+        pacCodeSecurityInfoLabel.numberOfLines = 0
+        pacCodeSecurityInfoLabel.textAlignment = .center
+        
+        
+        view.addSubview(pacCodeSecurityInfoLabel)
+        pacCodeSecurityInfoLabel.translatesAutoresizingMaskIntoConstraints = false
+        pacCodeSecurityInfoLabel.bottomAnchor.constraint(equalTo: pacCodeHStack.topAnchor, constant: -25).isActive = true
+        pacCodeSecurityInfoLabel.widthAnchor.constraint(equalTo: view.safeAreaLayoutGuide.widthAnchor, multiplier: 0.80).isActive = true
+        pacCodeSecurityInfoLabel.centerXAnchor.constraint(equalTo: view.safeAreaLayoutGuide.centerXAnchor).isActive = true
+    }
+    
+    func setupPacCodeTitleLabel() {
+        let pacCodeTitleLabel = UILabel()
+        pacCodeTitleLabel.textColor = .labelPrimary
+        pacCodeTitleLabel.font = UIFont.systemFont(ofSize: 35)
+        pacCodeTitleLabel.text = "PAC Code"
+        
+        view.addSubview(pacCodeTitleLabel)
+        pacCodeTitleLabel.translatesAutoresizingMaskIntoConstraints = false
+        pacCodeTitleLabel.bottomAnchor.constraint(equalTo: pacCodeSecurityInfoLabel.topAnchor, constant: -20).isActive = true
+        pacCodeTitleLabel.centerXAnchor.constraint(equalTo: view.safeAreaLayoutGuide.centerXAnchor).isActive = true
+    }
+    
+//    This is only in place as a placeholder for an icon if deemed necessary
+    func setupPlaceHolderIcon() {
```

Comment:
The UI has changed as shown below.
![Simulator Screen Shot - iPhone 11 Pro Max - 2020-03-09 at 18 22 36](https://user-images.githubusercontent.com/49562117/76262536-fb2ae280-6232-11ea-86a1-5087343c8eb9.png)

I thought it may be a good idea to have a sort of icon at the top of the vc to avoid making the page look so dull. I'm however completely down from removing it if you feel simplicity should be a priority.

Reply:
I like the graphic!